api/permissions.py

```diff
@@ -62,6 +62,14 @@ def has_permission(self, request, view):
         return InstructionRole.objects.filter(user=request.user).exists()
 
 
+class IsSupervisorOrInstructor(permissions.BasePermission):
```

Comment:
Dans la doc https://www.django-rest-framework.org/api-guide/permissions/
"Provided they inherit from rest_framework.permissions.BasePermission, permissions can be composed using standard Python bitwise operators. For example, IsAuthenticatedOrReadOnly could be written:"
ça devrait donc être possible, non ?
ça semble être arrivé avec DRF 3.9 https://www.django-rest-framework.org/community/3.9-announcement/#composable-permission-classes